test/unit/controllers/consolidations/ShutMUCRConfirmationControllerSpec.scala

```diff
@@ -19,17 +19,20 @@ package controllers.consolidations
 import controllers.ControllerLayerSpec
 import controllers.actions.AuthenticatedAction
 import controllers.storage.FlashKeys
-import models.ReturnToStartException
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import org.scalatestplus.mockito.MockitoSugar
 import play.api.http.Status
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
+import play.twirl.api.HtmlFormat
 import views.html.shut_mucr_confirmation
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class ShutMUCRConfirmationControllerSpec extends ControllerLayerSpec {
+class ShutMUCRConfirmationControllerSpec extends ControllerLayerSpec with MockitoSugar {
 
-  private val page = new shut_mucr_confirmation(main_template)
+  private val page = mock[shut_mucr_confirmation]
```

Comment:
I know that you're using this mock here and only on one test case, but in case that we will add more tests is better to use `beforeEach()` to mock behaviour of the page and `afterEach()` to reset the page.

Reply:
Agreed!